### ISSUE TITLE
Backspace on input is deleting selected options

### ISSUE THREAD

**Issue by vitornovo54** [moreta/vue-search-select#130]
When the select input is open and empty hitting backspace will delete all selected options.
Is there a method to overwrite this behaviour?

**Comment by tonila** [comment]
I also had this problem. Hitting backspace triggers @select with empty object. 

I had to use following workaround with all select events:

```
onSelect(selected) {
    if (!selected.value) return
    ....
}
```

**Comment by vitornovo54** [comment]
yes, eventually I had to removed it from the component and do a rebuild. 
But this looks like a nice workaround.
Thanks ;)
